src/lib.rs

```diff
@@ -463,12 +463,21 @@ fn execute_instr(input: InstrContext) -> Option<InstrEffects> {
 
     let clock = match sysvar_cache.get_clock() {
         Ok(clock) => (*clock).clone(),
-        Err(_) => Clock::default(),
+        Err(_) => {
+            let mut default_clock = Clock::default();
+            default_clock.slot = 10;
+            sysvar_cache.set_clock(default_clock.clone());
+            default_clock
+        }
     };
 
     let epoch_schedule = match sysvar_cache.get_epoch_schedule() {
         Ok(epoch_schedule) => (*epoch_schedule).clone(),
-        Err(_) => EpochSchedule::default(),
+        Err(_) => {
+            let default_epoch_schedule = EpochSchedule::default();
+            sysvar_cache.set_epoch_schedule(default_epoch_schedule.clone());
```

Comment:
Does it matter that these are clones? Do we need to retrieve these from the sysvar cache first (by calling sysvar_cache.get_clock()/sysvar_cache.get_epoch_boundary()) and pass these into the batch processor?